integration-tests-config-sample.yaml

```diff
@@ -1,5 +1,9 @@
 ---
 
+fixtures_repository:
+    # The GitHub repo containing fixtures used by test cases
+    url: https://github.com/foo/operators-integration-tests-fixtures
```

Comment:
Does this already exist, at least for your internal testing?

Reply:
No, it's supposed to contain ideally one branch per test case where each branch contains the minimum necessary to perform the test: in this case it would be an existing operator with at least one bundle and a commit (the head of the branch) would be the change to be tested.